src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ReactDOM from "react-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import Blogs from "./index";
+import { act } from "react-dom/test-utils";
+
+let container;
+
+beforeEach(() => {
```

Comment:
what happens if you delete lines 10 through 18? and line 8

Reply:
if I delete, what can I compare snapshot with?